docs/book/README.md

```diff
@@ -57,7 +57,7 @@ reliability.
 
 ## Resources
 
-- GitBook: [kubernetes-sigs.github.io/cluster-api](https://kubernetes-sigs.github.io/cluster-api)
+- GitBook: cluster-api.sigs.k8s.io
```

Comment:
```suggestion
- GitBook: [cluster-api.sigs.k8s.io](https://cluster-api.sigs.k8s.io)
```

Reply:
Doh. I had it this way and then thought maybe it was redundant...